### PR TITLE
Fix typos in source code

### PR DESCRIPTION
**Pull request by Sajjon**

### CODE REVIEW

File: Sources/Complex/Complex.swift [hunk 1]
```diff
@@ -313,7 +313,7 @@ extension Complex: Hashable {
     // Identify all numbers with either component non-finite as a single
     // "point at infinity".
     guard a.isFinite || b.isFinite else { return true }
-    // For finite numbers, equality is defined componentwise. Cases where
+    // For finite numbers, equality is defined component wise. Cases where
```
Comment (stephentyrone) [inline Sources/Complex/Complex.swift]: This one is correct as written.

Comment (Sajjon) [reply Sources/Complex/Complex.swift]: Ah, American English vs Brittish English. I'm used to the latter! Will make sure to remember this for the future :)

**Comment by Sajjon** [comment]
@stephentyrone I fixed the incorrect British English proposed spelling _"component wise"_ back to American English _"componentwise"_.

**Comment by stephentyrone** [comment]
Thanks @Sajjon!

**Pull request merged**